src/OVAL/probes/unix/linux/systemdunitdependency_probe.c

```diff
@@ -232,7 +239,7 @@ int systemdunitdependency_probe_main(probe_ctx *ctx, void *probe_arg)
 	if (dbus_conn == NULL) {
 		dbus_error_free(&dbus_error);
 		SEXP_t *msg = probe_msg_creat(OVAL_MESSAGE_LEVEL_INFO, "DBus connection failed, could not identify systemd units.");
-		probe_cobj_set_flag(probe_ctx_getresult(ctx), SYSCHAR_FLAG_ERROR);
+		probe_cobj_set_flag(probe_ctx_getresult(ctx), ctx->offline_mode == PROBE_OFFLINE_NONE ? SYSCHAR_FLAG_ERROR : SYSCHAR_FLAG_NOT_COLLECTED);
```

Comment:
Will this produce unknown rule result in case of `PROBE_OFFLINE_OWN` and failing to connect to DBus?

Reply:
It will produce unknown result. But I'm not sure if it's the desired behaviour in case it will be combined. See test `oval:0:tst:3` outcome in offline test.